tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.md

```diff
@@ -0,0 +1,25 @@
+Test 5-27 - Selenium Grid
+=======
+
+# Purpose:
+To verify that VIC works properly when a large grid of Selenium workers is deployed
+
+# References:
+[1 - Selenium Grid](https://github.com/SeleniumHQ/docker-selenium/blob/master/README.md)
+
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation
+
+# Test Steps:
+1. Deploy a new vCenter with a simple cluster
+2. Create a docker network for the selenium grid
```

Comment:
lgtm, May be I am missing it, where in the workflow docker network is created for the grid?